src/connector/src/sink/encoder/json.rs

```diff
@@ -216,7 +219,7 @@ fn datum_to_json_object(
                 }
                 json!(v_string)
             }
-            CustomJsonType::None | CustomJsonType::Bigquery => {
+            _ => {
```

Comment:
We should avoid using `_` and enumerate all variants so that in the future when we add more cases we can be notified whether the new variant has any special logic.